apps/cowswap-frontend/src/modules/accountProxy/pure/AccountCard/types.ts

```diff
@@ -0,0 +1,5 @@
+export enum AccountCardHoverBehavior {
+  NONE = 'none',      // No hover effects
```

Comment:
None is not in use, it's defaulted to `SELF` and only set to `PARENT` for `AccountItem` component.
Again, it's better to use a composition instead of configuration.